migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc

```diff
@@ -7,6 +7,9 @@ toc::[]
 
 This section describes considerations and strategies for planning your migration from {product-title} 3 to 4.
 
-// content:
-// https://bugzilla.redhat.com/show_bug.cgi?id=1968377
-// https://bugzilla.redhat.com/show_bug.cgi?id=1968352
+include::modules/migration-DNS-considerations.adoc[leveloffset=+2]
+
+[id="planning-considerations-3-4-additional-resources"]
+== Additional resources
```

Comment:
This resource belongs to the previous section, so give it a proper name like "Additional resources for DNS considerations".